external/cxxopts.hpp

```diff
@@ -1,6 +1,6 @@
 /*
 
-Copyright (c) 2014, 2015, 2016, 2017 Jarryd Beck
+Copyright (c) 2014-2022 Jarryd Beck
```

Comment:
@Meakk About cxxopts pathch, here is what I found.

1. There is currently no patch in cxxopts, so this PR should move forward withoout issues
2. I did had some troubles two years ago when reoworking the option system and opened some issues on cxxopts repo, but they were ultimaly closed as it was misuse on my part: https://github.com/jarro2783/cxxopts/issues?q=is%3Aissue+author%3Amwestphal+is%3Aclosed
3. At the time, there was a patch, but it may have been a local patch only and was never merged
4. Options rework is visible here, notic the assert to ensure we do not make the same mistake: https://github.com/f3d-app/f3d/pull/396/files#diff-7c425fb94271a2540a78bd3b67b26cf5f90dd107db21df60aaf3a57137b379f2
5. Some time later, you removed the assert and I approved it, because it was catching to wide a net and in some case this is supported.

The only unsupported case is vector<string> empty initialization, that we are currently avoiding.

So all good !

Reply:
Good